### ISSUE TITLE
Unexpected styling of ArgoCDDetailsWidget

### ISSUE THREAD

**Issue by anderoo** [RoadieHQ/backstage-plugin-argo-cd#45]
So I've ran into an 'interesting' style issue when using the ArgoCDDetailsWidget. This seems to only happen with this particular plugin and I'm not sure if the issue is caused by a conflict with other plugins or a conflict with the core styling of backstage.

## Expected Behavior

I would expect my Overview page containing the ArgoCDDetailsWidget to look like:

<img width="1680" alt="Screenshot 2021-03-02 at 17 12 36" src="https://user-images.githubusercontent.com/57673895/109686784-7f7c2100-7b7a-11eb-873a-a7f689a6c960.png">

## Current Behavior

Instead, it seems to modify a few different css files which are used by other components of Backstage. Most notably, it seems to add:

```
display: flex
align-items: center;
```

to the sidebar css class (Because of make-styles the class name isn't useful) and seems to adopt the css class of the logo for some elements within the table, for example the reason why the first row sync isn't viewable in the image below is it's  adding:

```
.jss29 {
    z-index: unset;
    position: relative;
    margin-top: -24px;
    margin-bottom: 48px;
}
```

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/57673895/109687110-d71a8c80-7b7a-11eb-8ab6-524301d2e893.png">

## Your Environment

- Plugin Version (version of this software e.g. v0.4.3): v0.3.3
- Backstage version (e.g. 0.1.1-alpha23): backstage/core: ^0.6.1
- Operating System and Version (e.g. Ubuntu 14.04): Running on a node-12 docker image
- Browser Information (e.g. chrome 85): Firefox

I'm not exactly sure where this issue is originating from, however it has only appeared since adding the ArgoCD plugin and only appears once the table is rendered. Have you come across this issue before?

**Comment by iain-b** [comment]
The version of material-ui/core was updated 15 days ago (snyk PR) to 4.11.3 and it's at 4.11.0 everywhere else I've looked. This could be the culprit. I'll try and repro your issue tomorrow and see if we can get to the bottom of it.

**Comment by anderoo** [comment]
> The version of material-ui/core was updated 15 days ago (snyk PR) to 4.11.3 and it's at 4.11.0 everywhere else I've looked. This could be the culprit. I'll try and repro your issue tomorrow and see if we can get to the bottom of it.

Yep, I'm thinking this may be the culprit. Going to test it out in the morning and post an update.

**Comment by anderoo** [comment]
Hey team. Update from my side, I've confirmed it's an issue with the material-ui/core version. ArgoCD plugin was on v4.11.3 whilst the rest of the plugins and core Backstage imports I were using were on 4.11.0. I'm happy to close this as the resolution will either updating the other packages or using v3.0.0 of this plugin until the other components have upgraded material-ui/core.

**Comment by iain-b** [comment]
Thanks @anderoo. I suppose from our side we'll need a process for managing the non-backstage package versions across our plugins (`backstage-cli versions` only covers backstage packages, naturally). In this case, if there's no vulnerability with 4.11.0 I think we might downgrade here to bring this plugin in line with the rest and with the upstream (the mui version doesn't appear to change often upstream).

**Comment by padraigobrien** [comment]
I have tested it at its previous value, "@material-ui/core": "^4.9.1", and it works as expected, I will submit a PR for it
